string object_name
geometry_msgs/Pose pose
string object_class

